src/styles/main.scss

```diff
@@ -1,3 +1,17 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables.scss";
+@import "./utils/mixins";
+
+@import "blocks/header.scss";
+@import "blocks/icons.scss";
+@import "blocks/menu.scss";
+@import "blocks/nav.scss";
+@import "blocks/page.scss";
+@import "blocks/top-bar.scss";
+@import "./blocks/main.scss";
```

Comment:
Why do you add files from the same directory differently? If you add files like 'blocks/...' you should use this syntax for all imports.